From 496a8d46b0695be78397285ef4872afa2bb96943 Mon Sep 17 00:00:00 2001
From: Trevor Gross <tg@trevorgross.com>
Date: Wed, 24 Jun 2026 01:03:24 -0400
Subject: [PATCH] ci: Add a very important "Hello World"

---
 ci/docker/x86_64-unknown-linux-gnu/Dockerfile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/ci/docker/x86_64-unknown-linux-gnu/Dockerfile b/ci/docker/x86_64-unknown-linux-gnu/Dockerfile
index ed9734ca2..3fa408311 100644
--- a/ci/docker/x86_64-unknown-linux-gnu/Dockerfile
+++ b/ci/docker/x86_64-unknown-linux-gnu/Dockerfile
@@ -9,3 +9,6 @@ RUN apt search linux-headers
 RUN ls /usr/src

 ENV PATH=$PATH:/rust/bin
+
+# Very important!
+RUN echo "Hello, world!"
